README.md

```diff
@@ -12,6 +12,12 @@ It main goal is to provide:
 
 ## Setup
 
+### Start
+
+Copy env variables `cp .env.development.sample .env.development`
```

Comment:
maybe it can also be done with make?
and also add `make setup` which does all necessary stuff

Reply:
It will not work :). When you copy env file you generally need to change it. (To correctly put username and password in Postgres). If you `make setup` it will fail as hanami could not find db (when running hanami db prepare). Then you need to correct you env file then run `make setup` again. Which will cause prompt from `cp` command (Do you want to overwrite the file?). Generally, it is bad practice to put not idempotent command in a makefile.

Reply:
Okay